src/main/java/hudson/plugins/ec2/win/EC2WindowsLauncher.java

```diff
@@ -64,11 +65,13 @@ protected void launchScript(EC2Computer computer, TaskListener listener) throws
 
                 OutputStream initGuard = connection.putFile(tmpDir + ".jenkins-init");
                 initGuard.write("init ran".getBytes(StandardCharsets.UTF_8));
+                initGuard.close();
```

Comment:
It would make more sense to change these code snippets to use try-with-resources like:

```
try (OutputStream o = ...) {
  ...
}
```